packages/server/package.json

```diff
@@ -18,7 +18,8 @@
         "mocha": "^9.2.2",
         "pino": "^7.11.0",
         "pino-pretty": "^7.6.1",
-        "uuid": "^3.3.2"
+        "uuid": "^3.3.2",
+        "pnpm": "^8.6.6"
```

Comment:
just one question? 
why was pnpm needed here?

Reply:
It's fixing https://github.com/hashgraph/hedera-json-rpc-relay/issues/1672. The `pnpm` was removed from the DockerFile and the `charts install` ci could not be able to build `packages/server` and that's why it started to fail recently. Adding `pnpm` as a dependency in `packages/server` will make `charts install` ci to succeed.

Reply:
Ohh, I see, thanks!! indeed the Test has passed on this branch. 👍 
Thank you!